src/boot/z_std_dma.c

```diff
@@ -217,46 +291,55 @@ const char* DmaMgr_GetFileNameImpl(uintptr_t vrom) {
 }
 
 const char* DmaMgr_GetFileName(uintptr_t vrom) {
-    const char* ret = DmaMgr_GetFileNameImpl(vrom);
+    const char* ret = DmaMgr_FindFileName(vrom);
 
     if (ret == NULL) {
         return "(unknown)";
     }
 
-    if (DmaMgr_CompareName(ret, "kanji") == 0 || DmaMgr_CompareName(ret, "link_animetion") == 0) {
+    if (DmaMgr_StrCmp(ret, "kanji") == 0 || DmaMgr_StrCmp(ret, "link_animetion") == 0) {
+        // Refuse kanji and link_animetion files. This may be related to these files being too large to be loaded all
+        // at once, however a NULL filename does not prevent them from being loaded.
```

Comment:
since

> a NULL filename does not prevent them from being loaded.

, I feel like

> Refuse kanji and link_animetion files.

is misleading. I'd suggest just removing that "refuse ... files" sentence, it's pretty clear what this does